flow/activities/flowable_core.go

```diff
@@ -628,7 +628,7 @@ func (a *FlowableActivity) startNormalize(
 	batchID int64,
 ) error {
 	ctx = context.WithValue(ctx, shared.FlowNameKey, config.FlowJobName)
-	logger := activity.GetLogger(ctx)
+	logger := shared.LoggerFromCtx(ctx)
```

Comment:
this should be removed since this method is no longer a top level activity. Caller should pass logger down